solutions/silver/cf-818F.mdx

```diff
@@ -0,0 +1,88 @@
+---
+id: cf-818F
+source: CF
+title: Level Generation
+author: Justin Ji
+---
+
+<Spoiler title="Hint 1">
+
+What is the upper bound for the number of bridges we can have?
+
+</Spoiler>
+
+<Spoiler title="Answer to Hint 1">
+
+The optimal construction would be to create a tree, which has $n - 1$ bridges.
```

Comment:
1. a direct answer would probably be better
2. you can't just say that :sob:

Reply:
wait wdym by "you can't just say that" lol

Reply:
surely at least give some intuition as to why it's the case

Reply:
hmm im not really sure how to lead into the idea of forming a tree without explicitly saying it

Reply:
well i guess that's the hard part of writing an editorial

Reply:
true lol

ngl i think just directly saying it might be more useful